torchgeo/datamodules/ftw.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""FTW datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+import torch
+
+from ..datasets import FieldsOfTheWorld
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+
+class FieldsOfTheWorldDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the FTW dataset.
+
+    .. versionadded:: 0.7
+    """
+
+    mean = torch.tensor([0, 0, 0, 0, 0, 0, 0, 0])
+    std = torch.tensor([3000, 3000, 3000, 3000, 3000, 3000, 3000, 3000])
```

Comment:
```suggestion
    mean = torch.tensor(0)
    std = torch.tensor(3000)
```
Much easier if all channels are the same